### PR TITLE
Use $shadow-color variable in $layout-shadow

### PR DESCRIPTION
**Pull request by popadicbranislav**
It is much easier to customise the shadow color if the `$layout-shadow` uses `$shadow-color` variable inside. This way if I change in my custom variables `$shadow-color`, I will get consistent shadows through the app.

**What kind of change does this PR introduce?** 

- [ ] Bugfix
- [ ] Feature
- [ ] Documentation
- [ ] Code style update
- [x] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** 

- [ ] Yes
- [x] No

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch (or `v[X]` branch)
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] It's been tested on a Cordova (iOS, Android) app
- [ ] It's been tested on an Electron app
- [x] Any necessary documentation has been added or updated [in the docs](https://github.com/quasarframework/quasar/tree/dev/docs)  or explained in the PR's description.

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to [start a new feature discussion](https://github.com/quasarframework/quasar/discussions/new?category=ideas-proposals) first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

**Comment by popadicbranislav** [comment]
I suppose I don't need to update the docs, since it looks like it is loading actual `variables.sass` file. Is it correct?

File: ui/src/css/variables.sass [hunk 1]
```diff
@@ -584,8 +584,8 @@ $toolbar-title-letter-spacing   : .01em !default
 $toolbar-title-padding          : 0 12px !default
 
 $layout-border                  : 1px solid $separator-color !default
-$layout-shadow                  : 0 0 10px 2px rgba(0,0,0,0.2), 0 0px 10px rgba(0,0,0,0.24) !default
-$layout-shadow-dark             : 0 0 10px 2px rgba(255,255,255,.2), 0 0px 10px rgba(255,255,255,.24) !default
+$layout-shadow                  : 0 0 10px 2px rgba($shadow-color,0.2), 0 0px 10px rgba($shadow-color,0.24) !default
```
Comment (MilosPaunovic) [inline ui/src/css/variables.sass:587]: I don't think that you can use `hex` values (`$shadow-color: #000`, `$dark-shadow-color: #FFF`) inside a `rgba` color definition.

Comment (popadicbranislav) [reply ui/src/css/variables.sass:587]: On the contrary, you can use hex values in variables this way. Here is working example in [codepen](https://codepen.io/popadicbranislav/pen/XWBwMxW)

Comment (MilosPaunovic) [reply ui/src/css/variables.sass:587]: You are correct, I apologize for the mistake.

**Review by MilosPaunovic — APPROVED** [review]

**Comment by rstoenescu** [comment]
Thanks for contributing!
This will go into Quasar 2.11.6

**Pull request merged**